inscriptions/ISic000707.xml

```diff
@@ -191,11 +196,16 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><roleName><supplied reason="lost"><expan><abbr>Imp</abbr><ex>erator</ex></expan></supplied></roleName> <roleName><supplied reason="lost"><expan><abbr>Caes</abbr><ex>ar</ex></expan></supplied></roleName> <persName type="emperor"><name type="praenomen"><supplied reason="lost">M</supplied>arcu<supplied reason="lost">s</supplied></name> <name type="nomen"><supplied reason="lost">Aurelius</supplied></name> <name type="cognomen"><supplied reason="lost">Antoninus</supplied></name>
+                    <lb n="2"/><name type="cognomen"><supplied reason="lost">Pius</supplied></name> <name type="cognomen"><supplied reason="lost">Aug</supplied>ustus</name></persName> <supplied reason="lost">et</supplied> <persName type="emperor"><name type="praenomen"><supplied reason="lost"><expan><abbr>L</abbr><ex>ucius</ex></expan></supplied></name> <name type="nomen"><supplied reason="lost">Aurelius</supplied></name> <name type="cognomen"><supplied reason="lost">Verus</supplied></name></persName> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
we would class all these as supernomina; although at the moment we're not generally adding the types on the names.